xbmc/addons/kodi-addon-dev-kit/UseMultiArch.cmake

```diff
@@ -0,0 +1,49 @@
+# - Multiarch support in object code library directories
```

Comment:
doesn't https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html do the same?